src/Controls/GlobeControls.js

```diff
@@ -759,26 +768,41 @@ class GlobeControls extends THREE.EventDispatcher {
         }
     }
 
-    handleZoom(event) {
+    handleZoom(event, dollyOrigin) {
         this.player.stop();
         CameraUtils.stop(this.view, this.camera);
 
-        this.updateTarget();
-        const delta = -event.delta;
-        this.dolly(delta);
+        var point = dollyOrigin ?? this.view.getPickingPositionFromDepth(event.viewCoords);        // mouse position
+        this.view.getPickingPositionFromDepth(null, pickedPosition);
+        var range = this.getRange(pickedPosition);
+        range *= (event.delta > 0 ?  this.zoomInScale :  this.zoomOutScale);
 
-        const previousRange = this.getRange(pickedPosition);
-        this.update();
-        const newRange = this.getRange(pickedPosition);
-        if (Math.abs(newRange - previousRange) / previousRange > 0.001) {
-            this.dispatchEvent({
-                type: CONTROL_EVENTS.RANGE_CHANGED,
-                previous: previousRange,
-                new: newRange,
-            });
+        if (point && (range > this.minDistance && range < this.maxDistance)) {  // check if the zoom is in the allowed interval
+            const camPos = this.getLookAtCoordinate().toVector3();
+
+            point = new Coordinates('EPSG:4978', point).as('EPSG:4326').toVector3();
+            if (camPos.x * point.x < 0) {     // Correct rotation at 180th meridian by using 0 <= longitude <=360
+                if (camPos.x - point.x > 180) { point.x += 360; } else if (point.x - camPos.x > 180) { camPos.x += 360; }
+            }
```

Comment:
It's a `lookAtCoordinate` issue, it's not support a correct rotation at 180th meridian by using `0 <= longitude <=360`?

Reply:
No it's for the lerp method (interpolation). At the 180th meridian, we have values going from -179 to 179 (-179, -180/180, 179). So when interpolating a value between, for example, -170 and 172, we will obtain a value of **1** instead of the expecting **-179**. 
To avoid that, I add 360 (to keep the same value modulo 360) and interpolate on positive values only. Then I substract the 360 to get the value back between -180 and 180.